third_party/terraform/utils/transport.go

```diff
@@ -120,14 +120,16 @@ func addQueryParams(rawurl string, params map[string]string) (string, error) {
 }
 
 func replaceVars(d TerraformResourceData, config *Config, linkTmpl string) (string, error) {
```

Comment:
While you're here, do you mind documenting what `replaceVars` does? Specifically, just that it expands `{{var}}` to a corresponding Terraform property, that it handles zone/region/project specially, and that it handles values starting with `%` specially by URL encoding them.

Honestly, this isn't a great place for documentation on how this works- I don't expect developers outside of our team to read a doc comment here when using MM- but I'd like to home the information _somewhere_ until we can find a better place to surface it.